third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/known-apps-registry.js

```diff
@@ -118,8 +125,14 @@ KnownAppsRegistry.prototype.getById = function(id) {
 };
 
 /**
+ * Similar to getById(), but performs a batch request for the list of given IDs
+ * at once.
+ *
+ * The result is returned asynchronously as a promise that, when resolved, will
+ * contain the same number of elements as |idList|, with each value containing
```

Comment:
I assume the entries also have the same order in the returned list?

Reply:
Good point. Yes, and I've reworded the comment to mention this.